From 5ca76fe473824f35328c3b77ca160bd4a62fd563 Mon Sep 17 00:00:00 2001
From: Victor Wan <victor.wan@amlogic.com>
Date: Fri, 10 Jan 2014 17:13:25 +0800
Subject: [PATCH 2853/5965] Revert "remove audio_data.o for new reserve memory"
 for merging

This reverts commit ab65fb52314ba371303ebe7f86d3d5d26d6748bc.
---
 drivers/amlogic/amaudio/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/amaudio/Makefile b/drivers/amlogic/amaudio/Makefile
index 30c47a1d129f..df67286ff3e5 100755
--- a/drivers/amlogic/amaudio/Makefile
+++ b/drivers/amlogic/amaudio/Makefile
@@ -13,7 +13,7 @@ endif
 $(obj)/audio_data.o:
 	@cp $(TOP_KBUILD_SRC)drivers/amlogic/amaudio/audio_data.z \
         drivers/amlogic/amaudio/audio_data.o
-#amlogic_amaudio-objs += audio_data.o
+amlogic_amaudio-objs += audio_data.o
 endif
 
 obj-$(CONFIG_AMAUDIO) += amlogic_amaudio.o
-- 
2.19.0

